.github/workflows/ci.yaml

```diff
@@ -40,7 +40,10 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
-
+      - uses: actions/setup-go@v5
```

Comment:
we should do the same for the other jobs I think (e2e / metallb e2e)

Reply:
e2e they use  only `make e2etests` and then  `$(GINKGO) -v $(GINKGO_ARGS) --timeout=3h ./e2etests -- $(TEST_ARGS)`  so  we should control the ginkgo version probably by doing `go install -mod=vendor github.com/onsi/ginkgo/v2/ginkgo` (need to check if we do vendor). Not sure the version of go is important or not

Probably same for metallb2 e2e. 

But we can add it

Reply:
go install compiles gingko, so yes, in order to have less drifts as possible I think it make sense to pin there too.
The ginkgo version is pinned already.